src/simulate.jl

```diff
@@ -57,12 +57,12 @@ function simulate(p, biomass; start::Int64=0, stop::Int64=500, use::Symbol=:nons
       extspecies = Int[]
       #isext = falses(S)
 
-      function condition(t,y,integrator)
+      function condition(u,t,integrator)
```

Comment:
This arg flip is why it fixes it. That was one of the 4.0 changes.

Reply:
Oh ok! Thanks for your help!